runtime/src/lib.rs

```diff
@@ -511,13 +518,13 @@ construct_runtime!(
 		Historical: session_historical::{Module} = 20,
 		//Cennzx: crml_cennzx::{Module, Call, Storage, Config<T>, Event<T>} = 21,
 		// TODO: these should all be in one module
-		// SyloGroups: sylo_groups::{Module, Call, Storage} = 22,
-		// SyloE2EE: sylo_e2ee::{Module, Call, Storage} = 23,
-		// SyloDevice: sylo_device::{Module, Call, Storage} = 24,
-		// SyloInbox: sylo_inbox::{Module, Call, Storage} = 25,
-		// SyloResponse: sylo_response::{Module, Call, Storage} = 26,
-		// SyloVault: sylo_vault::{Module, Call, Storage} = 27,
-		// SyloPayment: sylo_payment::{Module, Call, Storage} = 28,
+		SyloGroups: sylo_groups::{Module, Call, Storage} = 22,
```

Comment:
will need to implement the trait for the Runtime for each of these traits also which is why CI build is failling

Reply:
done